hey-discount/src/main/webapp/resources/js/View.js

```diff
@@ -0,0 +1,589 @@
+                block.querySelector(".review-text-2").textContent = ad.reviews[1].reviewText;
+                block.querySelector(".user-review-3").textContent = ad.reviews[2].username;
+                block.querySelector(".date-review-3").textContent = ad.reviews[2].date.getMonth() + '/' + ad.reviews[2].date.getDate() + '/' + ad.reviews[2].date.getFullYear();
+                block.querySelector(".review-text-3").textContent = ad.reviews[2].reviewText;
+                block.querySelector(".user-review-4").textContent = ad.reviews[3].username;
+                block.querySelector(".date-review-4").textContent = ad.reviews[3].date.getMonth() + '/' + ad.reviews[3].date.getDate() + '/' + ad.reviews[3].date.getFullYear();
+                block.querySelector(".review-text-4").textContent = ad.reviews[3].reviewText;
+                block.querySelector(".review-5").style.borderColor = 'white';
+            }
+
+            if (ad.reviews.length == 5) {
+                block.querySelector(".user-review-1").textContent = ad.reviews[0].username;
+                block.querySelector(".date-review-1").textContent = ad.reviews[0].date.getMonth() + '/' + ad.reviews[0].date.getDate() + '/' + ad.reviews[0].date.getFullYear();
+                block.querySelector(".review-text-1").textContent = ad.reviews[0].reviewText;
+                block.querySelector(".user-review-2").textContent = ad.reviews[1].username;
+                block.querySelector(".date-review-2").textContent = ad.reviews[1].date.getMonth() + '/' + ad.reviews[1].date.getDate() + '/' + ad.reviews[1].date.getFullYear();
+                block.querySelector(".review-text-2").textContent = ad.reviews[1].reviewText;
+                block.querySelector(".user-review-3").textContent = ad.reviews[2].username;
+                block.querySelector(".date-review-3").textContent = ad.reviews[2].date.getMonth() + '/' + ad.reviews[2].date.getDate() + '/' + ad.reviews[2].date.getFullYear();
+                block.querySelector(".review-text-3").textContent = ad.reviews[2].reviewText;
+                block.querySelector(".user-review-4").textContent = ad.reviews[3].username;
+                block.querySelector(".date-review-4").textContent = ad.reviews[3].date.getMonth() + '/' + ad.reviews[3].date.getDate() + '/' + ad.reviews[3].date.getFullYear();
+                block.querySelector(".review-text-4").textContent = ad.reviews[3].reviewText;
+                block.querySelector(".user-review-5").textContent = ad.reviews[4].username;
+                block.querySelector(".date-review-5").textContent = ad.reviews[4].date.getMonth() + '/' + ad.reviews[4].date.getDate() + '/' + ad.reviews[4].date.getFullYear();
+                block.querySelector(".review-text-5").textContent = ad.reviews[4].reviewText;
+            }
+
+        }
+        return block;
+    }
```

Comment:
This method is more then 100 lines long 😱 
Let's try to refactor it - move comments creation etc to separate methods